web/models/email.ex

```diff
@@ -54,4 +57,15 @@ defmodule HexWeb.Email do
   def order_emails(emails) do
     Enum.sort_by(emails, &[not &1.primary, not &1.public, not &1.verified, -&1.id])
   end
+
+  defp validate_verified_email_exists(changeset, field, opts) do
```

Comment:
Please move this to `validations.ex` we should avoid accessing the repo inside model modules.